Library/Homebrew/cmd/fetch.rb

```diff
@@ -64,7 +65,7 @@ def fetch
   def fetch_bottle?(f)
     return true if ARGV.force_bottle? && f.bottle
     return false unless f.bottle && f.pour_bottle?
-    return false if ARGV.build_from_source? || ARGV.build_bottle?
+    return false if FormulaInstaller.build_formula_from_source?(f) || ARGV.build_bottle?
```

Comment:
I see what you mean. And that's needed for correct behavior for `install --build-bottle`, too. I amended the PR to do so. Look good now?

Tests out for `fetch`:

```
$ brew fetch --build-bottle --deps analog xz
Fetching: analog, xz, pkg-config, libpng, freetype, fontconfig, jpeg, libtiff, gd
==> Downloading https://mirrors.ocf.berkeley.edu/debian/pool/main/a/analog/analog_6.0.orig.tar.gz
######################################################################## 100.0%
Downloaded to: /Library/Caches/Homebrew/analog-6.0.tar.gz
SHA256: 31c0e2bedd0968f9d4657db233b20427d8c497be98194daf19d6f859d7f6fcca
==> Downloading https://fossies.org/linux/misc/xz-5.2.2.tar.gz
######################################################################## 100.0%
Downloaded to: /Library/Caches/Homebrew/xz-5.2.2.tar.gz
SHA256: 73df4d5d34f0468bd57d09f2d8af363e95ed6cc3a4a86129d2f2c366259902a2
==> Downloading https://homebrew.bintray.com/bottles/pkg-config-0.29.1.mavericks.bottle.tar.gz
Already downloaded: /Library/Caches/Homebrew/pkg-config-0.29.1.mavericks.bottle.tar.gz
SHA256: a27ddccd93c19c3e492c6dba674140340b9ac18253066b5a91933d7ca4dd0992
==> Downloading https://homebrew.bintray.com/bottles/libpng-1.6.21.mavericks.bottle.tar.gz
######################################################################## 100.0%
Downloaded to: /Library/Caches/Homebrew/libpng-1.6.21.mavericks.bottle.tar.gz
SHA256: 22b421d6d8034884fb0b697001b0965cb19f3b3a32c0c619ead86250e6f2a40d
==> Downloading https://homebrew.bintray.com/bottles/freetype-2.6.3.mavericks.bottle.tar.gz
Already downloaded: /Library/Caches/Homebrew/freetype-2.6.3.mavericks.bottle.tar.gz
SHA256: ecfb1b9f4dcce361d4c40367dc205842411cbcbb6628238da571aff84caf05c3
```